test/integration/controllers/notification.spec.js

```diff
@@ -41,24 +60,38 @@ describe('Notification controller', () => {
   })
 
   describe(`GET ${endpointUrl}`, () => {
-    it('should get user\'s notifications and count them', async () => {
+    it("should get user's notifications and count them", async () => {
       const response = await app.get(endpointUrl).set('Authorization', `Bearer ${accessToken}`)
 
-      console.log(response.body)
-
       expect(response.statusCode).toBe(200)
       expect(response.body.count).toBe(1)
       expect(response.body.items[0]).toMatchObject({
-        _id:testNotification._id,
-        createdAt:expect.any(String),
-        updatedAt:expect.any(String),
+        _id: testNotification._id,
+        createdAt: expect.any(String),
+        updatedAt: expect.any(String),
         ...testNotificationData
       })
     })
 
     it('should throw UNAUTHORIZED', async () => {
       const response = await app.get(endpointUrl)
-  
+
+      expectError(401, UNAUTHORIZED, response)
+    })
+  })
+
+  describe(`POST ${endpointUrl}`, () => {
+    it('should create notification', async () => {

```

Comment:
In your test file, you're testing the post method, however, you didn't add a route in the router file. Add post method in the router/notification.js

Reply:
Also update the router/index.js file